mcux/mcux-sdk/components/rpmsg/fsl_adapter_rfimu.c

```diff
@@ -1544,16 +1544,16 @@ static void HAL_ImuTaskDeinit(uint8_t link)
 #ifndef __ZEPHYR__
         (void)OSA_EventDestroy((osa_event_handle_t)ImuQ13FlagsRef);
         (void)OSA_TaskDestroy(ImuTaskHandleCpu13);
-#endif
         imu_task_flag &= ~(1U << link);
+#endif
```

Comment:
Seems like this change should really just be:

```
#if defined(CPU2)
...
#else 
#ifndef __ZEPHYR__
// all the code
#endif 
#endif 
```

Reply:
I have create an issue, how to attach issue to the PR.
issue link: https://github.com/zephyrproject-rtos/zephyr/issues/80598

Reply:
@GaofengZhangNXP , have you looked into the suggestion made by @dleach02 above?